tests/tests_common_test.go

```diff
@@ -220,3 +220,15 @@ func expectUserCan(sars *authv1.SubjectAccessReviewSpec) {
 			sars.ResourceAttributes.Subresource, sars.ResourceAttributes.Name, sars.ResourceAttributes.Group,
 			sars.ResourceAttributes.Version, sars.ResourceAttributes.Namespace))
 }
+
+func expectUserCannot(sars *authv1.SubjectAccessReviewSpec) {
+	sar, err := coreClient.AuthorizationV1().SubjectAccessReviews().Create(ctx, &authv1.SubjectAccessReview{
+		Spec: *sars,
+	}, metav1.CreateOptions{})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
+	ExpectWithOffset(1, sar.Status.Allowed).To(BeFalse(),
```

Comment:
I think checking for the `Denied` field is safer

Reply:
Actually not
`Denied` is option and can be flaky even when `Allowed` is false